Themis/Views/Exercises/ExerciseView.swift

```diff
@@ -100,6 +100,12 @@ struct ExerciseView: View {
         }
         .buttonStyle(NavigationBarButton())
     }
+    private var submissionDueDateNotOver: Bool {
```

Comment:
Can we change this variable to `submissionDueDateOver` and just negate it when needed so that the code is easier to read?

Reply:
Yes, makes sense. Thank you!